pages/[user].js

```diff
@@ -63,6 +64,14 @@ const UserPage = () => {
         </p>
       )}
 
+      <p>
+        <UserFeedStatus isGone={user.isGone}
+                        isPrivate={user.isPrivate}
+                        isProtected={user.isProtected}
+                        isRestricted={user.isRestricted}
+                        type={user.type}/>
```

Comment:
I think replacing all the props here with simple `{...user}` should be fine. I know it will be not as explicit as it is now, but it will look nicer.